doc/nrf/test_and_optimize/optimizing/power_nrf91.rst

```diff
@@ -17,6 +17,8 @@ See the `Online Power Profiler user guide`_ for a basic overview of the tool and
 
 The following sections describe how you can use Online Power Profiler and other power profiling tools to optimize the power consumption in your nRF9160 DK.
 
+See the :ref:`cellular_psm` for more information on different power saving modes.
```

Comment:
My only nit is that this is going to link the full document for all the power saving techniques, and not just power saving modes.